packages/terraform-modules-public/gcp/example/README.md

```diff
@@ -299,6 +299,176 @@
 
 17. Troubleshooting
 
-    If your new validator isn't signing, check and make sure that both the proxy and the validator are synced, and that the enode variable for the proxy is set correctly in terraform.tfvars.
+    If your new validator isn't signing, check the following:
+    
+    * Make sure that both the proxy and the validator are synced.  You can verify this in the geth console:
+    ```console
+    docker exec -it geth geth attach
+    eth.syncing
+    ```
+    * Ensure that the validator signer key is unlocked on the validator:
+    ```console
+    docker exec -it geth geth attach
+    personal
+    ```
+    * Ensure that the proxy has >100 peers
+    ```console
+    docker exec -it geth geth attach
+    admin.peers.length
+    ```
+    * Check that the enode variable for the proxy is set correctly in terraform.tfvars.
+    * Verify network connectivity from the validator to the proxy on tcp/30503
+
+18. Attestation Service
+
+    First generate a new account to use for the attestation signer.
+
+    ```console
+    celocli account:new
+    ```
+
+    Use these values for the `attestation_signer_accounts` attributes:
+
+        * account_addresses
+        * private_keys
+        * account_passwords
+        
+    Put these into terraform.tfvars.
+
+    Now, on a system which has access to the attestation_signer private key, generate a proof of possession for that key as follows:
+
+    ```bash
+    #!/bin/bash
+    set -x
+
+    ######
+    # use this script on an attestation signer tx-node to generate a proof of possession, needed for key rotation
+
+    CELO_IMAGE=us.gcr.io/celo-org/celo-node:f4bda6d027f0af6c68ae4d146abe791739ae6cba
```

Comment:
What do you think of `us.gcr.io/celo-org/geth:1.1.0`?

Reply:
Right.  Of course.